makefile.shade

```diff
@@ -8,4 +8,4 @@ k-standard-goals
 custom-goals
 
 #initialize if='Directory.Exists("src")'
-  exec program='dotnet' commandline='run -p ../../tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs' workingdir='${kestrelSrcDir}'
```

Comment:
I don't think we should do the internal renamespacering for stuff in the `tools` folder. If we publish packages from here, we're already doing something very wrong.